jupyros/ros2/key_input.py

```diff
@@ -0,0 +1,94 @@
+## Keyboard input for Jupyter Ros 2
+"""
+Author:         Luigi Dania
+Email:          Luigi@dobots.nl
+Github:         https://github.com/ldania
+
+Company:        Dobots
+Company Repo:   https://github.com/dobots/ 
+
+
+"""
+
+import sys
+sys.path.append("./../jupyter-ros")
+import jupyros.ros2 as jr2
+from ipyevents import Event 
```

Comment:
For good practice, it's a good idea to remove libraries which are not used and that will prevent installation errors. However, if you do require new libraries, we also need to include them in the *setup.py* file under `install_requires=[...]`.